FifthLesson/VectorProgram/VectorProgram/View/ConsoleView.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace VectorProgram.View
+{
+    public class ConsoleView : IConsoleView
+    {
+        public void PressKeyToContinue() => Console.ReadKey(true);
```

Comment:
Maybe **WaitForAnyKeyPress** it is better name

Reply:
Yes, it is better. I fixed.